src/entity/schema.ts

```diff
@@ -316,7 +316,7 @@ export const ActionSchema = z.object({
     cardAmount: z.number().int().positive().optional(),
     cards: z.array(CardSchema).optional(),
     nominatedPlayer: PlayerSchema.optional(),
-    nominatedColor: z.nativeEnum(CardColor).optional(),
+    nominatedAmount: z.number().int().min(1).max(3).optional(),

```

Comment:
Warum die color entfernen?
```suggestion
    nominatedColor: z.nativeEnum(CardColor).optional(),
```